{{ cookiecutter.repo_name }}/Makefile

```diff
@@ -120,8 +120,11 @@ pull_base_image: docker_login populate_aws_id
 	docker pull ${AWS_BASE_IMAGE} \
         && docker tag ${AWS_BASE_IMAGE} analytics-base:${IMAGE_VERSION}
 
+jupyter_notebook_config.json:
+	$(PYTHON_INTERPRETER) create_notebook_config.py
```

Comment:
Could we please add a comment saying that you'll be requested to set a password here (for jupyter notebook)? And perhaps a note that you will need to remember that password for whenever you launch the container and jupyter?

Reply:
Could we make a JIRA ticket to update the README to reflect this process?

Reply:
I had that thought as well, but for some reason didn't say it. Upon revisiting that thought, I incline toward saying that updating the readme is in scope for this PR.

Reply:
Changing my approval to a change request to reflect my change of mind about the readme.